### PR TITLE
TMDB API Connection & show the now playing movies

### LINKED ISSUE #34: Connection to Movie Database (TMDB API)
## Expected Behaviour

Implement the connection to TMDB
https://developer.themoviedb.org/reference/movie-now-playing-list

## Checklist

- [x]  Signup on tmdb page
- [x]  get secret key and store it in a .env file
- [x]  share the .env file with your classmates via discord
- [x]  make sure the .env file is on gitIgnore
- [x]  fetch now playing movies from tmdb by using the secret key

### Hints

    - Refer to the TMDB webpage for using the API

### PR DESCRIPTION
**Pull request by focusedXmind**
## Checklist

Confirm you have completed the following actions prior to submitting this PR.

- [X] There is an existing issue report for this PR.
- [X] I have created a feature branch.
- [X] My changes have been committed.
- [X] I have pushed my changes to the branch.

## Title

TMDB API Connection & show the now playing movies

## Description

The problem is that we don't want to manually write the data of movies now playing in the movie theaters so we want to use the TMDB API to get the data dynamically from their database.

## Issue Resolution

Using the TMDB API to fetch the movies that are now playing in the movie theaters.

Closes #34 

## Proposed Changes

_List your proposed changes below._

- signed up at TMDB
- got the API Key
- shared the key with class in discord
- added the .env file on gitignore
- fetched the new movies (services/tmdb.ts)

## New or Changed Features

EXTENDED FEATURES:
to see if the fetching is working right, i already made it possible to show the new movies. So I added a movie-Type (types/movie.ts) and a nowplayingmovies Component (nowplayingmovies.tsx). It would be sad if this approach has to be deleted because we use it in the future anyways right? :D

### CODE REVIEW

File: src/components/genre.tsx
Comment (louziferr) [inline src/components/genre.tsx:1]: Nice, but not requested as part of this issue. Part of issue #40 .

File: src/components/nowplayingmovies.tsx
Comment (louziferr) [inline src/components/nowplayingmovies.tsx:1]: Nice, but not requested as part of this issue. Refer to #19

File: src/services/tmdb.ts [hunk 1]
```diff
@@ -0,0 +1,19 @@
+import axios from "axios";
+
+const apiKey = import.meta.env.VITE_TMDB_API_KEY;
+console.log("API Key:", apiKey); // Add this line to check if the API key is correctly loaded
+const baseUrl = "https://api.themoviedb.org/3";
+
+export const fetchNowPlayingMovies = async () => {
```
Comment (louziferr) [inline src/services/tmdb.ts]: Please rename to getNowPlayingMovies

**Review by louziferr — CHANGES_REQUESTED** [review]
Please remove genre.tsx and nowplayingmovies.tsx from commit.

File: src/components/genre.tsx
Comment (focusedXmind) [reply src/components/genre.tsx:1]: this was also my issue ;)

File: src/services/tmdb.ts (hunk 1 above)
Comment (focusedXmind) [reply src/services/tmdb.ts]: check

**Review by louziferr — APPROVED** [review]
Nice!

**Pull request merged**